perfdash/metrics-downloader.go

```diff
@@ -19,19 +19,25 @@ package main
 import (
 	"context"
 	"fmt"
+	"io"
 	"io/ioutil"
-	"k8s.io/klog"
 	"path"
 	"path/filepath"
 	"sort"
 	"strconv"
 	"strings"
 	"sync"
 
+	"k8s.io/klog"
+
```

Comment:
nit: most Go code will just have 2 groups of import blocks; stdlib and other.

Reply:
Let your editor manage all this and we can use the same import management rules.